NineChronicles.Headless.Executable/Program.cs

```diff
@@ -188,7 +188,9 @@ static void ConfigureSentryOptions(SentryOptions o)
             string? sentryDsn = "",
             [Option(Description = "Trace sample rate for sentry")]
             double? sentryTraceSampleRate = null,
-            [Ignore] CancellationToken? cancellationToken = null
+            [Ignore] CancellationToken? cancellationToken = null,
+            [Option(Description = "Enable rate limiting to control rate of requests.")]
+            bool? rateLimit = null
```

Comment:
Though it isn't a rule, how about ordering the `Ignore` parameter to the last?
```suggestion
            [Option(Description = "Enable rate limiting to control rate of requests.")]
            bool? rateLimit = null,
            [Ignore] CancellationToken? cancellationToken = null
```